e2eAIOK/deltatuner/deltatuner/utils/config.py

```diff
@@ -10,4 +10,6 @@ class DeltaTunerType(str, enum.Enum):
 TRANSFORMERS_MODELS_TO_SSF_TARGET_MODULES_MAPPING.update({
     "llama": ["q_proj", "v_proj"],
     "mpt": ["Wqkv","out_proj","up_proj","down_proj"]
-})
\ No newline at end of file
+})
+
+BEST_MODEL_STRUCTURE_NAME = "best_model_structure.txt"
```

Comment:
I also suggest that we do not fix the best_model_structure filename.

Reply:
Let me explain the logic for you, there are two ways to assign the `best_model_structure`
- Through local file `BEST_MODEL_STRUCTURE_NAME` which is saved after searching, usually used in `from_pretrained` function
- Through the input args `DeltaTunerArguments.best_model_structure`, usually used in `optimize` function before fine-tuning process

The above two way is complementary to each other, and provide more flexibility to users. I think you misunderstand the above two cases, I have change the file name from `BEST_MODEL_STRUCTURE_NAME` to `BEST_MODEL_STRUCTURE_DEFAULT_NAME` in commit [e7cbb47](https://github.com/intel/e2eAIOK/pull/414/commits/e7cbb4731a1e8e872ec1f3b1bcb692415e76c61a), which can help users to better understand the variable usage.